### PR TITLE
Create spinner

### PR DESCRIPTION
**Pull request by fortanza**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
OK for me

**Comment by LoicBrassart** [comment]
No SM validation

**Review by Caroline-CD — DISMISSED** [review]
OK for me :+1:

**Review by Hacene010 — DISMISSED** [review]
Seems ok

File: src/Containers/SearchCards/SearchCards.jsx [hunk 1]
```diff
@@ -16,17 +21,41 @@ const Cards = () => {
...
   useEffect(() => {
+    setLoading(true);
     axios
       .get(
         `https://api.magicthegathering.io/v1/cards?&Page-Size=1&name=${needleName}&power=${needlePower}&toughness=${needleToughness}&type=${needleType}&colors=${needleColor}&rarity=${needleRarity}&set=${needleSet}`
       )
       .then(({ data }) => {
         setMagicCards(data.cards);
+
+        setLoading();
```
Comment (LoicBrassart) [inline src/Containers/SearchCards/SearchCards.jsx]: For consistency's sake, write `setLoading(false)` instead

File: src/Containers/SearchCards/SearchCards.jsx [hunk 2]
```diff
@@ -16,17 +21,41 @@ const Cards = () => {
...
+      })
+      .catch(() => {
+        console.error('Plz fix your call, or set up your internet');
+        setLoading();
```
Comment (LoicBrassart) [inline src/Containers/SearchCards/SearchCards.jsx]: For consistency's sake, write `setLoading(false)` instead

File: src/Containers/SearchCards/SearchCards.jsx [hunk 3]
```diff
@@ -16,17 +21,41 @@ const Cards = () => {
...
       .get(
         `https://api.magicthegathering.io/v1/cards?&Page-Size=1&name=${needleName}&power=${needlePower}&toughness=${needleToughness}&type=${needleType}&colors=${needleColor}&rarity=${needleRarity}&set=${needleSet}`
       )
       .then(({ data }) => {
         setMagicCards(data.cards);
+
+        setLoading();
+      })
+      .catch(() => {
+        console.error('Plz fix your call, or set up your internet');
```
Comment (LoicBrassart) [inline src/Containers/SearchCards/SearchCards.jsx:35]: No `console.*`

File: src/Containers/SearchCards/SearchCards.jsx [hunk 4]
```diff
@@ -41,6 +70,7 @@ const Cards = () => {
         <p className="searchError">Oups, no result for this request...</p>
       )}
     </>
+
```
Comment (LoicBrassart) [inline src/Containers/SearchCards/SearchCards.jsx]: Useless empty line

File: src/UI/Spinner/Spinner.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import './Spinner.css';
+
+const Spinner = () => {
+  return (
+    <div className={['loader', 'loader--fan-out'].join(' ')}>
+      <div className={['item', 'item-1'].join(' ')} />
+      <div className={['item', 'item-2'].join(' ')} />
+      <div className={['item', 'item-3'].join(' ')} />
+      <div className={['item', 'item-4'].join(' ')} />
+      <div className={['item', 'item-5'].join(' ')} />
```
Comment (LoicBrassart) [inline src/UI/Spinner/Spinner.jsx]: Why complexify class names declarations ? 
`{['loader', 'loader--fan-out'].join(' ')}` can be replaced by `"loader loader--fan-out"`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please clean these errors

File: src/Containers/SearchCards/SearchCards.jsx (hunk 3 above)
Comment (fortanza) [reply src/Containers/SearchCards/SearchCards.jsx:35]: problem fixed

File: src/Containers/SearchCards/SearchCards.jsx (hunk 4 above)
Comment (fortanza) [reply src/Containers/SearchCards/SearchCards.jsx]: problem fixed

File: src/UI/Spinner/Spinner.jsx (hunk 5 above)
Comment (fortanza) [reply src/UI/Spinner/Spinner.jsx]: problem fixed

File: src/Containers/SearchCards/SearchCards.jsx (hunk 2 above)
Comment (fortanza) [reply src/Containers/SearchCards/SearchCards.jsx]: problem fixed

File: src/Containers/SearchCards/SearchCards.jsx (hunk 1 above)
Comment (fortanza) [reply src/Containers/SearchCards/SearchCards.jsx]: problem fixed

**Review by Caroline-CD — APPROVED** [review]
Modifications ok

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**